Gemfile

```diff
@@ -30,7 +30,7 @@ gem 'browser'
 gem 'charlock_holmes', '~> 0.7.7'
 gem 'iso-639'
 gem 'chewy', '~> 5.2'
-gem 'cld3', '~> 3.4.2'
+# gem 'cld3', '~> 3.4.2'
```

Comment:
This seems to be in use, did you have any trouble with it?

Reply:
You can view this, https://github.com/rubygems/bundler/issues/5655. It gives solution for both mac and linux systems.